Automattic/wp-calypso
# ci(e2e) Use real chrome version in UA string

#### Background

We send a custom UserAgent identifier in e2e tests. This identifier includes `wp-e2e-tests` as the system information.

In order to include Chrome version in the User Agent we had a script that, when preparing CI to run e2e tests, runs `chrome --version` and saves the output to a file `./.chromedriver_version`).

That script was working on CircleCI, but it didn't get ported to TeamCity. However, because we had accidentally commited `./.chromedriver_version` to the repo in the past, it still works.

#### Changes proposed in this Pull Request

* Delete `./.chromedriver_version` from the repo
* Instead of reading from `./.chromedriver_version`, use the package `@testim/chrome-version` to get the version of Chrome installed in the system.

#### Testing instructions

* Verify e2e tests pass
* Inspect the e2e logs and check the user agent sent by Chrome includes its version.
